File: Sandbox.Modules.FSharp/General/DemoModule.fs
namespace Sandbox.Modules.FSharp.General

open Sandbox.Core

[<SandboxModule("F# Demo Module", "fs-demo", "This module is a simple test of integrating the Visual Basic module collection with the module service.")>]
[<ModuleTags("general", "sandbox", "testing")>]
type DemoModule() =
   inherit SandboxModule()
   override b.Execute() = b.SendResponse(SandboxEventType.None, "Hello World from an F# module!")

File: Sandbox.Modules.VisualBasic/General/DemoModule.vb
Imports Sandbox.Core
Namespace Sandbox.Modules.VisualBasic.General
    <SandboxModule("Visual Basic Demo Module", "vb-demo", "This module is a simple test of integrating the Visual Basic module collection with the module service.")>
    <ModuleTags("general", "sandbox", "testing")>
    Public Class DemoModule
        Inherits SandboxModule

        Protected Overrides Sub Execute()
            SendResponse(SandboxEventType.None, "This is a test of integrating a Visual Basic module!")
        End Sub
    End Class
End Namespace